### PR TITLE
Improve displaying events

### PR DESCRIPTION
**Pull request by lukaw3d**
| Before | After |
| --- | --- |
| [link with all event types](https://50733e59.oasis-explorer.pages.dev/mainnet/emerald/tx/0x481380a29028d51ee19a06ef0cae56d5719b0f9465e2adba8a26fce9961f2f4c) | [link with all event types](https://9acdaf47.oasis-explorer.pages.dev/mainnet/emerald/tx/0x481380a29028d51ee19a06ef0cae56d5719b0f9465e2adba8a26fce9961f2f4c) |
| 50733e59 oasis-explorer pages dev_mainnet_emerald_tx_0x481380a29028d51ee19a06ef0cae56d5719b0f9465e2adba8a26fce9961f2f4c | 9acdaf47 oasis-explorer pages dev_mainnet_emerald_tx_0x481380a29028d51ee19a06ef0cae56d5719b0f9465e2adba8a26fce9961f2f4c |

### CODE REVIEW

File: src/app/pages/TransactionDetailPage/index.tsx [hunk 1]
```diff
@@ -240,155 +240,8 @@ export const TransactionDetailView: FC<{
...
 
-          <dt>{t('common.block')}</dt>
-          <dd>
-            <BlockLink scope={transaction} height={transaction.round} />
-          </dd>
-
-          <dt>{t('common.type')}</dt>
-          <dd>
-            <RuntimeTransactionLabel method={transaction.method} />
-          </dd>
```
Comment (lukaw3d) [inline src/app/pages/TransactionDetailPage/index.tsx:251]: (just easier demo)

File: src/oasis-nexus/api.ts [hunk 2]
```diff
@@ -520,10 +520,183 @@ export const useGetRuntimeEvents: typeof generated.useGetRuntimeEvents = (
...
+                },
+                evm_log_name: '',
+                round: 4172572,
+                tx_hash: null,
+                type: 'consensus_accounts.withdraw',
+              },
+            ],
+            is_total_count_clipped: false,
+            total_count: 7,
+          } as any
```
Comment (lukaw3d) [inline src/oasis-nexus/api.ts:667]: (temporary data to demo all event types)

**Comment by lukaw3d** [comment]
For reference, from Don:
1. We don’t want to show any raw code? If we, I’d probably place it in an accordion.
2. Do you think it would be possible to add iconography to the event type?
3. Mobile layout

---
1. raw bytes next to parsed events probably aren't useful at all, and might be confusing
2. TODO: will try to reuse TokenTransferIcon
3. TODO

**Comment by csillag** [comment]
> For reference, from Don:
> 
> 1. We don’t want to show any raw code? If we, I’d probably place it in an accordion.
> 2. Do you think it would be possible to add iconography to the event type?
> 3. Mobile layout
> 
> 1. raw bytes next to parsed events probably aren't useful at all, and might be confusing
> 2. TODO: will try to reuse TokenTransferIcon
> 3. TODO

Is the plan to cover the TODO items in this same PR, or in a subsequent one?

**Review by csillag — APPROVED** [review]
Reviewed, LGTM, except the linting errors.

(And I'm not sure if we want to leave in the debug data)

**Comment by lukaw3d** [comment]
I'll rather merge this and make a separate PR

**Comment by lukaw3d** [comment]
(without debug data)

**Pull request merged**